packages/orchestrator-ui-components/src/components/WfoBadges/WfoWebsocketStatusBadge/WfoHeaderWebsocketBadge.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { useTranslations } from 'next-intl';
+
+import { EuiFlexGroup, EuiText } from '@elastic/eui';
+
+import { WfoHeaderBadge } from '@/components';
+import { useWithOrchestratorTheme } from '@/hooks';
+import { useOrchestratorTheme } from '@/hooks/useOrchestratorTheme';
+import { WfoBoltFill, WfoBoltSlashFill } from '@/icons';
+import { orchestratorApi } from '@/rtk';
+import { useStreamMessagesQuery } from '@/rtk/endpoints/streamMessages';
+
+import { getStyles } from './styles';
+
+export const WfoHeaderWebsocketBadge = () => {
+    const dispatch = useDispatch();
+    const { disconnectedStyle } = useWithOrchestratorTheme(getStyles);
+
+    const t = useTranslations('main');
+    const { theme } = useOrchestratorTheme();
+    const { data: websocketConnected } = useStreamMessagesQuery();
+
+    const reconnect = useCallback(() => {
+        dispatch(orchestratorApi.util.resetApiState());
+    }, [dispatch]);
+
+    return !websocketConnected ? (
+        <EuiFlexGroup gutterSize="s" alignItems="center">
+            <EuiText size="s" color={theme.colors.accentText}>
+                {t('websocketDisconnectedShort')}
+            </EuiText>
```

Comment:
This text might have insufficient contrast with its background:
![image](https://github.com/user-attachments/assets/c66c0474-e32f-4612-bc13-2b389e84f0dd)

Also wouldn't make a tooltip sense here? The icon is red enough to scream for user attention. A simple hover would reveal the reason. Similar behavior as the bagde with the 17000+ failed tasks